ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -618,6 +618,7 @@ public static KsqlRestApplication buildApplication(final KsqlRestConfig restConf
     final String ksqlServerId = ksqlConfig.getString(KsqlConfig.KSQL_SERVICE_ID_CONFIG);
     updatedRestProps.putAll(
         MetricCollectors.addConfluentMetricsContextConfigs(ksqlServerId, kafkaClusterId));
+    updatedRestProps.put(KsqlConfig.KSQL_PROPERTIES_OVERRIDES_DENYLIST, "ksql.service.id");
```

Comment:
if this was already specified, wouldn't this overwrite the previous value?

Reply:
Yes, you are right :sweat_smile:. I changed it.